packages/components/aside/src/aside.css

```diff
@@ -11,7 +11,7 @@
 	min-height: 100vh;
 	width: var(--ts-aside);
 	background-color: var(--ts-color-white);
-	box-shadow: 0 0 3px var(--ts-shadow-color);
+	box-shadow: 0 0 3px var(--ts-color-slate-dark);
```

Comment:
Shouldn't we change the `ts-shadow-color` to slate-dark?

Reply:
The shadow in the aside is different with the current shadow in the board.
The slate-dark is only for aside, And the light-gray is for the common shadow.